src/NuGet.Core/NuGet.Packaging/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,5 @@
+static NuGet.Packaging.Rules.AnalysisResources.InvalidUndottedFrameworkInDependencyGroupsWarning.get -> string
```

Comment:
scope creep:  It's just a guess, but I guess that `AnalysisResources` are public only so tests can access the message text without using `InternalsVisibleTo`. These strings are not useful to customers using the SDK, because we could change the `{x}` placeholders at any time.

What do you think about making a new resx file that's internal, so we stop getting these public API changes just because we're adding messages?

Reply:
I'm gonna treat this as scope creep. Maybe we can create an issue to find a better solution for AnalysisResources in general?